libs/libdict/Makefile

```diff
@@ -0,0 +1,43 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=libdict
```

Comment:
This should look like:

PKG_NAME:=libdict
PKG_SOURCE_DATE:=2018-05-20
PKG_SOURCE_VERSION:=95f6d27f277932272578b2554104e1e4414cfaac
PKG_RELEASE=1